app/controllers/book_controller.rb

```diff
@@ -0,0 +1,14 @@
+class BookController < ApplicationController
+  include Wor::Paginate
+  include DeviseTokenAuth::Concerns::SetUserByToken
+
+  def index
+    render_paginated Book.all, limit: 2
+  end
+
+  def show
+    render json: Book.find(params[:id])
+  end
+
+  before_action :authenticate_user!
```

Comment:
This will be use for almost all you controller, better to put it in your ApiController

Reply:
Ok, I will

Reply:
You didn't... But with the previous comment you will be able to understand what is the ApiController :)